blueocean-core-js/package.json

```diff
@@ -83,10 +83,20 @@
     "jest": "19.0.2",
     "jest-cli": "19.0.2",
     "jest-junit": "1.5.1",
+    "karma": "1.3.0",
+    "karma-browserify": "5.1.0",
+    "karma-chrome-launcher": "2.0.0",
+    "karma-junit-reporter": "1.1.0",
+    "karma-mocha": "1.2.0",
+    "karma-mocha-reporter": "2.2.0",
+    "karma-phantomjs-launcher": "1.0.2",
```

Comment:
the karma stuff can go... was removed in a previous PR

Reply:
👍  done